### ISSUE TITLE
Give menu i.s.o. number for which parameter to delete

### ISSUE THREAD

**Issue by thoni56** [thoni56/c-xrefactory#57]
The current "give number of argument to remove" is not the best approach. A list of formal argument names would be so much better.

**Comment by shanemd** [comment]
Possibly a different feature request, but it would also be helpful to be able to remove a parameter 'at point' instead of having to place point on the function name.

**Comment by thoni56** [comment]
Yes, it would seem that traversing "up" to find the relevant function should not be a huge problem. However, since `c-xrefactory` doesn't actually build an AST it is not that straight forward. And I'm the first to admit to not knowing enough of how, or if, that would be possible.
